src/Sylius/Component/Shipping/spec/Resolver/CompositeMethodsResolverSpec.php

```diff
@@ -38,7 +37,7 @@ function it_is_initializable()
 
     function it_implements_composite_methods_resolver_interface()
```

Comment:
This function name should really be changed to represent the change in interface away from a composite methods resolver to a methods resolver.